flow-diagrams/subplat/sequence-diagrams/fullCheckoutSuccessSignedIn.mmd

```diff
@@ -0,0 +1,44 @@
+sequenceDiagram
+    participant B as Browser
+    participant P as Payments
+    participant F as FxA
+    participant D as DB
+    participant S as Stripe
+    participant C as CMS
+
+    B->>P: Navigate to /vpn?interval=monthly
+    Note over B,P: As FxA signed in user without Payments session
+    P->>F: Validate access token
+    F->>P: Access Token is valid
+    P->>F: Fetch Session by FxA UID
+    F->>P: No session found
+    alt createSessionWithAccessToken.mmd
```

Comment:
Miss using the `alt` tag here a little just to indicate this flow is also covered in `sub/createSessionWithAccessToken.mmd`